Settings Activity Crash

**Describe the bug**
Settings Activity Crash

**Logs**

`2019-07-21 12:54:19.678 29361-29361/com.codingblocks.cbonlineapp.debug E/AndroidRuntime:     at android.view.LayoutInflater.rInflate(LayoutInflater.java:866)
        at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:824)
        at android.view.LayoutInflater.inflate(LayoutInflater.java:515)
        at io.github.inflationx.viewpump.ViewPumpLayoutInflater.inflate(ViewPumpLayoutInflater.java:55)
        at android.view.LayoutInflater.inflate(LayoutInflater.java:423)
        at android.view.LayoutInflater.inflate(LayoutInflater.java:374)
        at androidx.appcompat.app.AppCompatDelegateImpl.setContentView(AppCompatDelegateImpl.java:469)
        at androidx.appcompat.app.AppCompatActivity.setContentView(AppCompatActivity.java:140)
        at com.codingblocks.cbonlineapp.activities.SettingsActivity.onCreate(SettingsActivity.kt:32)
        at android.app.Activity.performCreate(Activity.java:7163)
        at android.app.Activity.performCreate(Activity.java:7154)
        at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1288)
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3087)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3248)
        at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:78)
        at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:108)
        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:68)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1945)
        at android.os.Handler.dispatchMessage(Handler.java:106)
        at android.os.Looper.loop(Looper.java:193)
        at android.app.ActivityThread.main(ActivityThread.java:7009)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:537)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
     Caused by: java.lang.UnsupportedOperationException: Can't convert to ComplexColor: type=0x1
        at android.content.res.ResourcesImpl.loadComplexColorForCookie(ResourcesImpl.java:1114)
        at android.content.res.ResourcesImpl.loadComplexColorFromName(ResourcesImpl.java:968)
        at android.content.res.ResourcesImpl.loadColorStateList(ResourcesImpl.java:1069)
        at android.content.res.Resources.loadColorStateList(Resources.java:1053)
        at android.content.res.TypedArray.getColorStateList(TypedArray.java:568)
        at android.graphics.drawable.GradientDrawable.updateStateFromTypedArray(GradientDrawable.java:1325)
        at android.graphics.drawable.GradientDrawable.inflate(GradientDrawable.java:1241)
        at android.graphics.drawable.DrawableInflater.inflateFromXmlForDensity(DrawableInflater.java:142)
        at android.graphics.drawable.Drawable.createFromXmlInnerForDensity(Drawable.java:1332)
        at android.graphics.drawable.Drawable.createFromXmlInner(Drawable.java:1321)
        at android.graphics.drawable.LayerDrawable.inflateLayers(LayerDrawable.java:279)
        at android.graphics.drawable.LayerDrawable.inflate(LayerDrawable.java:194)
        at android.graphics.drawable.DrawableInflater.inflateFromXmlForDensity(DrawableInflater.java:142)
        at android.graphics.drawable.Drawable.createFromXmlInnerForDensity(Drawable.java:1332)
        at android.graphics.drawable.Drawable.createFromXmlForDensity(Drawable.java:1291)
        at android.content.res.ResourcesImpl.loadDrawableForCookie(ResourcesImpl.java:845)
        	... 51 more`

**Smartphone Info:**
|               | |
|---------------|-|
|Device         | Oneplus 7 Pro |
|Android Version| 9.0  |

**Would you like to work on the issue?**
YES

--- a/app/src/main/res/drawable/seekbar_track.xml
+++ b/app/src/main/res/drawable/seekbar_track.xml
@@ -5,7 +5,7 @@
         android:gravity="center_vertical|fill_horizontal">
         <shape
             android:shape="rectangle"
-            android:tint="@*android:color/material_grey_800">
+            android:tint="#424242">
             <size android:height="4dp" />
             <corners android:radius="5.0dip" />
             <solid android:color="#40000000" />

